tests/test_data/packages/small_fake_with_build_deps/pyproject.toml

```diff
@@ -1,8 +1,4 @@
 [build-system]
-requires = [
-    "setuptools==68.1.2",
-    "wheel==0.41.1",
-    "fake_static_build_dep"
```

Comment:
I wonder if having a trailing comma after the last item here would make the formatter to the right thing and keep the list. Worth checking...

Reply:
It does not, but that would be lovely.

Reply:
I added a [Feature Request](https://github.com/un-ts/toml-tools/issues/80) in the plugin project.